tests/emulator.ts

```diff
@@ -100,3 +106,9 @@ export function attachDatums(
   emulator.datumTable = { ...emulator.datumTable, ...datums };
   return emulator;
 }
+
+export function generateHash(): KeyHash | ScriptHash {
```

Comment:
```suggestion
export function generateHash(length: number): KeyHash | ScriptHash {
```
Since we have 32-bytes hash for datum and other things as well.